tests/unit_tests/blas/include/test_common.hpp

```diff
@@ -460,6 +460,13 @@ typename std::enable_if<std::is_integral<fp>::value, bool>::type check_equal(fp
     return (x == x_ref);
 }
 
+template <typename fp>
+bool check_equal_ptr(sycl::queue queue, fp *x, fp x_ref, int error_mag) {
```

Comment:
can we give a more descriptive name to indicate `x` is actually on device?

Reply:
I'm happy to change the name, but this function will work fine whether x is on host or device or shared.